Classes/Player/PlayerController.swift

```diff
@@ -248,6 +248,10 @@ class PlayerController: NSObject, Player {
     }
     
     func isLive() -> Bool {
+        if let playbackType = currentPlayer.playbackType, playbackType == "LIVE" {
```

Comment:
Where does this `"LIVE"` come from? It's not used anywhere else? 
Better make it a constant, even fileprivate in PlayerController.